_config.yml

```diff
@@ -0,0 +1 @@
+theme: jekyll-theme-architect
```

Comment:
It looks like you were experimenting with the design of the web page. Did you think the Architect theme looked better?